statsmodels/graphics/regressionplots.py

```diff
@@ -148,11 +151,12 @@ def plot_fit(results, exog_idx, y_true=None, ax=None, **kwargs):
         ax.plot(x1, y_true[x1_argsort], 'b-', label='True values')
     title = 'Fitted values versus %s' % exog_name
 
-    prstd, iv_l, iv_u = wls_prediction_std(results)
+    _, iv_l, iv_u = wls_prediction_std(results)
```

Comment:
AFAICS, this can also go under `if vlines is True`. It is not needed of it is not True